Makefile

```diff
@@ -3,6 +3,17 @@ IMG ?= quay.io/ecosystem-appeng/dbaas-e2e-test-harness:latest
 DIR := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
 OUT_FILE := "$(DIR)dbaas-e2e-test-harness"
 
+install-tools:
+	go install golang.org/x/tools/cmd/goimports@latest
+	go install github.com/mgechev/revive
+
+fmt: install-tools
```

Comment:
This target makes me `go get github.com/mgechev/revive`, and then creates a lot of indirect dependency entries in my mod file that you don't have. Why is that?

A few of them are:
> 	github.com/mgechev/dots v0.0.0-20210922191527-e955255bf517 // indirect
> 	github.com/mgechev/revive v1.2.1 // indirect
> 	github.com/mitchellh/go-homedir v1.1.0 // indirect
>

Reply:
Good catch. "go install github.com/mgechev/revive" causes to run "go get" which would download the dependencies. Changed to use specific version "go install github.com/mgechev/revive@latest" to fix this issue.